dotcom-rendering/src/web/lib/getCountryCode.ts

```diff
@@ -1,46 +1,5 @@
-import { getCookie, getLocale, isString, storage } from '@guardian/libs';
+import { getLocale } from '@guardian/libs';
 import type { CountryCode } from '@guardian/libs';
-import { countries } from './countryCodes';
-
-const COUNTRY_CODE_KEY = 'GU_geo_country';
-const COUNTRY_CODE_KEY_OVERRIDE = 'gu.geo.override';
-/*
-	Memoizes the location
- */
-let locale: CountryCode | null;
-
-/*
-	Can be used to override country code and sets it in localStorage
- */
-export const overrideCountryCode = (countryCode: CountryCode): void => {
-	storage.local.set(COUNTRY_CODE_KEY_OVERRIDE, countryCode);
-	locale = countryCode;
-};
-
-export const isCountryCode = (code: unknown): code is CountryCode => {
-	if (!isString(code)) return false;
-	if (countries.includes(code as CountryCode)) return true;
-	return false;
-};
-
-/*
-	This method can be used as a non async way of getting the country code
-	after getLocaleCode has been called.
- */
-export const getCountryCodeSync = (): CountryCode | null => {
```

Comment:
All of this is seemingly done by `@guardian/libs` already by https://github.com/guardian/csnx/blob/main/libs/%40guardian/libs/src/locale/getLocale.ts

Reply:
The last place that relied on `getCountryCodeSync` was the Lotame `init` script, which was removed in #3017 :

https://github.com/guardian/dotcom-rendering/pull/3017/files#diff-361662365c3498da8ac478810d45f93c533944db91483c085f2792472fe0d427